### DISCUSSION TITLE
Tamagui compatibility?

### DISCUSSION THREAD

**Discussion by redbar0n** [nandorojo/moti#144] [Ideas]
I just wanted to know if/how Moti could work with Tamagui.dev ?

It looks attractive for those of us who need full SSR support and thus cannot use Dripsy atm.

**Comment by redbar0n** [comment]
I’ve only seen this:
https://news.ycombinator.com/item?id=29326204

but it indicates that @natew is positive.

**Comment by nandorojo** [comment]
Any hold up for Moti support would likely be on the Tamagui side, since Moti itself works on any React Native primitive with a `style` prop.

**Comment by natew** [comment]
Yea definitely in support, been taking holidays off but will be back on things soon enough. 

If anyone sets up a simple example repo with a breaking case I’ll take a look.
